hubmap/frontend/src/components/Experiments.js

```diff
@@ -0,0 +1,115 @@
+            <StyledTableCell align="right">Data type</StyledTableCell>
+            <StyledTableCell align="right">Tissue</StyledTableCell>
+            <StyledTableCell align="right">Uploaded on</StyledTableCell>
+            <StyledTableCell align="right">Genes </StyledTableCell>
+            <StyledTableCell align="right">Proteins</StyledTableCell>
+            <StyledTableCell align="right"># Cells</StyledTableCell>
+            <StyledTableCell align="right"># Unique barcode </StyledTableCell>
+            <StyledTableCell align="right"># Reads </StyledTableCell>
+            <StyledTableCell align="right"># Reads Aligned </StyledTableCell>
+            <StyledTableCell align="right"># Images </StyledTableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {this.props.experiments ?
+            this.props.experiments.map(experiment => (
+              <StyledTableRow key={experiment.id}>
+                <StyledTableCell component="th" scope="row">{experiment.id}</StyledTableCell>
+                <StyledTableCell component="th" scope="row">{experiment.subclass.model}</StyledTableCell>
+                <StyledTableCell align="right">{experiment.institution.name}</StyledTableCell>
+                <StyledTableCell align="right">{experiment.data_type.name}</StyledTableCell>
+                <StyledTableCell align="right">{experiment.tissue.name}</StyledTableCell>
+                <StyledTableCell align="right">{experiment.creation_time}</StyledTableCell>
+                {experiment.genes !== undefined ?
+                  <StyledTableCell align="right">{experiment.genes.map((gene) => gene.hugo_symbol).join(',')}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.proteins !== undefined ?
+                  <StyledTableCell align="right">{experiment.proteins.map((protein) => protein.name).join(',')}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.cell_count ?
+                  <StyledTableCell align="right">{experiment.cell_count}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.Unique_barcode_count ?
+                  <StyledTableCell align="right">{experiment.Unique_barcode_count}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.read_count_total ?
+                  <StyledTableCell align="right">{experiment.read_count_total}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.read_count_aligned ?
+                  <StyledTableCell align="right">{experiment.read_count_aligned}</StyledTableCell>
+                  : <StyledTableCell align="right"></StyledTableCell>}
+                {experiment.image_count ?
```

Comment:
Here and above, I think you can either get rid of the ternary entirely, or move it inside the component: Since there's no difference in the wrapper, you can move the ternary inside, and that will make this whole thing much shorter. Like:
```jsx
<StyledTableCell align="right">{experiment.image_count ? experiment.image_count : null}</StyledTableCell>
```
or just:
```jsx
<StyledTableCell align="right">{experiment.image_count}</StyledTableCell>
```

Reply:
sure, will do, appreciate the suggestion.

Reply:
I implemented the first one, since the this material-ui table auto-detects column value types and since I have, for now, for prototype, this is not implemented. also this Experiments table is not used, but is here for checking different tables we could possibly use. We can switch to this, if this is what we prefer to use.